assets/src/edit-story/components/panels/layerStyle.js

```diff
@@ -49,10 +49,32 @@ function LayerStylePanel({ selectedElements, onSetProperties }) {
   useEffect(() => {
     setState({ opacity });
   }, [opacity]);
-  const handleSubmit = (evt) => {
+
+  const updateProperties = useCallback(() => {
     onSetProperties(state);
+  }, [onSetProperties, state]);
+
+  const handleSubmit = (evt) => {
+    updateProperties();
     evt.preventDefault();
   };
+
+  const handleNumberChange = (property) => (value) =>
+    setState((originalState) => {
+      const update = {
+        ...originalState,
+        [property]: isNaN(value) || value === '' ? '(auto)' : parseFloat(value),
```

Comment:
Should this be a placeholder `(auto)` instead of setting the actual value as `(auto)`? Wouldn't this cause issues if removing the whole value (e.g. clearing the field to type a new value)?

Reply:
It was a copy/paste from what the panel was using before my changes, I did not try different values here.

Reply:
Can't see this on `master`. If you look below then the opacity did not have `(auto)` set previously.

Reply:
Just tested and this is definitely causes issues. Mind pointing out the place where you copied it from so that we could remove it from there as well?

Reply:
Ah! Get the issue. It seems like a typo from `textStyle.js`. Let me remove that.